test/db/cmd/cmd_rop

```diff
@@ -2376,3 +2376,244 @@ Register dependencies:
 
 EOF
 RUN
+
+NAME=search rop gadgets given the detailed gadget size (=4)
+FILE=bins/arm/crackme.arm32.bin
+CMDS=<<EOF
+e asm.arch=arm
```

Comment:
Can you please print some of the gadgets in the test as well. So we have confirmation they actually match the attributes searched for?